dev/feature/3424dfb.unknown-connection-sqlexecutequery/bigquery2.yml

```diff
@@ -0,0 +1,21 @@
+---
+type: "script"
+data:
+  name: "db.bigquery.2"
+  description: "test db.bigquery connection with standard select query"
+  parameters: []
+  actions:
+  - number: 1
+    type: "sql.executeQuery"
+    name: "Action1"
+    description: "Select data from table1 in the iesi schema"
+    component: ""
+    condition: ""
+    iteration: ""
+    errorExpected: "N"
+    errorStop: "N"
+    parameters:
+    - name: "query"
+      value : "select * from iesi.table1;"
+    - name: "connection"
+      value : "db.bigquery.2"
```

Comment:
Are these files added to the repo as test or example scripts for documentation?

Reply:
These are the scripts to test the implementation